pkg/promutil/prometheus.go

```diff
@@ -118,24 +119,63 @@ func (p *PrometheusCollector) Describe(_ chan<- *prometheus.Desc) {
 
 func (p *PrometheusCollector) Collect(metrics chan<- prometheus.Metric) {
 	for _, metric := range p.metrics {
-		metrics <- createMetric(metric)
+		metrics <- metric
 	}
 }
 
-func createMetric(metric *PrometheusMetric) prometheus.Metric {
-	gauge := prometheus.NewGauge(prometheus.GaugeOpts{
-		Name:        *metric.Name,
-		Help:        "Help is not implemented yet.",
-		ConstLabels: metric.Labels,
-	})
+func toMetrics(metrics []*PrometheusMetric) []prometheus.Metric {
```

Comment:
do we want to commit this function and the benchmark test code if this isn't being used anymore?

Reply:
I noted this in the description,

> I'll cleanup the dead code from the old code path before merging just wanted to show what was benchmarked first.

Now that I have a test that proves it's functionally the same I deleted the old code and the bench test.